test/newElection.js

```diff
@@ -109,4 +116,18 @@ contract('NewElection', function() {
     });
   });
 
+  it("allows a voter to become registered", function() {
+    var currentElection;
```

Comment:
I think we can refactor our tests further to have these repeated vars appear just once  on lines 4-7 that all it blocks will be able to compare. Let's keep this as it is for the moment however, we can do this in another refactor branch